scripts/set-new-screen.jlink

```diff
@@ -0,0 +1,4 @@
+mem 0xe002 1
+w1 0xe002 1
```

Comment:
can you also add scripts to set it back to the old one? on raw pcb's it's easy to switch screens.

probably not necessary for the chip as you can't switch that :)

Reply:
done!